.github/workflows/release.yml

```diff
@@ -0,0 +1,99 @@
+name: Build and Release
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build-armv7:
+    name: Build Linux ARMv7 (32-bit)
+    runs-on: ubuntu-24.04-arm
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install cross compiler
+        run: sudo apt-get update && sudo apt-get install -y gcc-arm-linux-gnueabihf
+      - name: Build
+        run: |
+          make clean
+          make wake-on-arp CC=arm-linux-gnueabihf-gcc
+          mv wake-on-arp wake-on-arp-linux-armv7
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: wake-on-arp-linux-armv7
+          path: wake-on-arp-linux-armv7
+
+  build-armv7-raspbian-bullseye:
+    name: Build Linux ARMv7 (Raspbian Bullseye)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build in Bullseye Docker
+        uses: addnab/docker-run-action@v3
+        with:
+          image: debian:bullseye
+          options: -v ${{ github.workspace }}:/workspace
+          run: |
+            apt-get update
+            apt-get install -y build-essential gcc-arm-linux-gnueabihf make
```

Comment:
Not to pick, but whats the difference between building an ARMv7 executable under Debian or Ubuntu here?